cmd/fyne/internal/commands/get.go

```diff
@@ -8,7 +8,6 @@ import (
 	"path/filepath"
 	"strings"
 
-	"fyne.io/fyne/v2/cmd/fyne/internal/util"
```

Comment:
This (plus others later in) cannot be right?
Files don't compile if the import isn't used - so either this didn't compile before (actions should have caught it) or it doesn't compile now...

Reply:
I'm not sure how it's possible that the actions were passing before and after this change - can you explore?

Reply:
They are right. That's how mock work, by intercepting the access to io operation and allowing them to get verified on how they are called and what they return. The package util file will do the default redirection, when no mock are in place while the package util mock test will do that redirection when running the tests.

Reply:
And yes, it does compile :-)

Reply:
My apologies, I completely missed that is how it works. Very slick :)